src/components/MapFacetContents.js

```diff
@@ -75,6 +78,33 @@ class MapFacetContents extends React.Component<MapFacetContentsDefaultProps, Map
     };
   }
 
+  /**
+   * Fetches the latitude and longitude from the SearchFacetBucket value.
```

Comment:
I would change "fetches" to "gets" or something since "fetch" in JavaScript implies a network call...